frontend/lib/src/features/password_reset/components/reset_password_page.dart

```diff
@@ -0,0 +1,193 @@
+        } else if (message == 'New password does not match.') {
+          formKey.currentState?.fields['password_confirmation']
+              ?.invalidate(message);
+        } else {
+          formKey.currentState?.fields['token']?.invalidate(message);
+        }
+      }
+    }
+
+    return SafeArea(
+      child: Scaffold(
+        body: AuthHeader(
+          child: Padding(
+            padding: const EdgeInsets.fromLTRB(25, 0, 25, 150),
+            child: GetX<LoginController>(
+              builder: (_) => FormBuilder(
+                key: formKey,
+                autovalidateMode: AutovalidateMode.disabled,
+                onChanged: () {
+                  formKey.currentState!.save();
+
+                  // when password is invalidated on the onSubmit,
+                  // revalidate the password field when typing again
+                  formKey.currentState?.fields['password']
+                      ?.validate(focusOnInvalid: false);
+
+                  // set button enabled state
+                  buttonController.setButtonEnabled =
+                      formKey.currentState!.fields['password']!.isValid &&
+                          formKey.currentState!.fields['password_confirmation']!
+                              .isValid &&
+                          formKey.currentState!.fields['token']!.isValid;
+                },
+                child: Column(
+                  children: [
+                    Column(
+                      children: [
+                        Container(
+                          margin: const EdgeInsets.only(bottom: 40),
+                          alignment: Alignment.centerLeft,
+                          child: Text(
+                            'Create New Password',
+                            style: Theme.of(context).textTheme.titleMedium,
+                          ),
+                        ),
+                        Container(
+                          margin: const EdgeInsets.only(bottom: 14),
+                          child: Column(
+                            children: [
+                              InputField(
+                                name: 'password',
+                                label: 'Password',
+                                inputType: TextInputType.visiblePassword,
+                                obscureText: true,
+                                validator: FormBuilderValidators.required(),
+                              ),
+                              const SizedBox(
+                                height: 36,
+                              ),
+                              InputField(
+                                name: 'password_confirmation',
+                                label: 'Confirm Password',
+                                inputType: TextInputType.visiblePassword,
+                                obscureText: true,
+                                validator: FormBuilderValidators.required(),
```

Comment:
add validator here
password confirmation must be the same with password
refer to change password page